src/random.js

```diff
@@ -0,0 +1,18 @@
+const faker = require('faker')
+const randomInt = require('lodash/random')
+
+const random = () => {
+  const generators = Object.keys(random)
+  const generator = generators[randomInt(generators.length - 1)]
```

Comment:
My last commit also fixes a bug here...This could potentially return undefined since `randomInt` returns numbers up to and including the given upper limit.